src/core/stream/adaptation/types.ts

```diff
@@ -105,6 +105,18 @@ export interface IWaitingMediaSourceReloadPayload {
   stayInPeriod : boolean;
 }
 
+export interface INeedsBufferFlushPayload {
+  /** Relative resuming position after a track change */
+  relativeResumingPosition : number;
+  /** `true` if the user manually set relativeResumingPosition value.
```

Comment:
It's here `false` if the user manually set `relativeResumingPosition`, you mixed it up I think

Reply:
Oh yes I inverted both